server/parsec/components/postgresql/invite.py

```diff
@@ -1733,8 +1733,6 @@ async def complete(
                 return InviteCompleteBadOutcome.AUTHOR_NOT_FOUND
             case CheckDeviceBadOutcome.USER_REVOKED:
                 return InviteCompleteBadOutcome.AUTHOR_REVOKED
-        if current_profile != UserProfile.ADMIN:
```

Comment:
And the change is not needed in memory because we are using the logic in `is_greeter_allowed` ?